lib/run/options.js

```diff
@@ -354,13 +353,27 @@ module.exports = function (options, callback) {
     // allow insecure file read by default
     options.insecureFileRead = Boolean(_.get(options, 'insecureFileRead', true));
 
-    config.get(options, { loaders: configLoaders, command: 'run' }, function (err, result) {
+    // sanitize environment and globals
+    !options.environment && (options.environment = {});
+    !options.globals && (options.globals = {});
+
+    let commonOptions = _.pick(options, ['postmanApiKey']);
+
+    async.mapValuesSeries(options, (value, name, cb) => {
```

Comment:
why in series?

Reply:
Because the logic contains prompting for a passkey.
So once the postmanApiKey is decrypted for collection, it should not be asked for environment.

Reply:
So is it okay or should I find an alternate way?